db/schema.rb

```diff
@@ -80,7 +80,7 @@
   create_table "users", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
     t.string "name"
     t.string "email"
-    t.boolean "gender"
+    t.integer "gender"
```

Comment:
giá trị default mô rồi, và ko cho phép null nữa nhé
`tìm hiểu lại enum in rails ` đi